pyproject.toml

```diff
@@ -25,7 +25,7 @@ dependencies = [
     "httpx",
     "numexpr",
     "numpy",
-    "morecantile>=3.1,<4.0",
+    "morecantile>=4.0,<5.0",
```

Comment:
IMO it's safer to restrict the lower version to 4.0.
application using titiler might (like titiler 🙈) may not have pinned morecantile to specific versions